TrashMob/client-app/src/components/MainEvents.tsx

```diff
@@ -46,6 +47,8 @@ export const MainEvents: React.FC<MainEventsDataProps> = (props) => {
                 dispEvent.eventDate = mobEvent.eventDate;
                 dispEvent.eventTypeId = mobEvent.eventTypeId;
                 dispEvent.name = mobEvent.name;
+                dispEvent.creator = mobEvent.createdByUserId;
```

Comment:
I don't think this is the right field. This is a guid representing the user id to the system, not the User Name. We will need to add the User Name to the Display Event that is returned if we want to display this to others.

Reply:
I'll add to your PR to get this field corrected.

Reply:
Fixed